### PR TITLE
DM-25034 make an install-external-data butler subcommand

### PR DESCRIPTION
**Pull request by n8pease**
help looks like this:

```
$ butler install-external-data --help
Usage: butler install-external-data [OPTIONS] REPO

  Install extenral data

  REPO is the URI or path to an existing data repository root or
  configuration file.

Options:
  --source TEXT            Source of external data
  --tract INTEGER          Tract identifier
  --visit-ccd INT INT ...  Visit and CCD of jointcal data to ingest.
  -h, --help               Show this message and exit.
```

### CODE REVIEW

**Comment by TallJimbo** [comment]
Is this what we were calling `butler import` before?  This name feels like the name of a very ci_hsc-specific script.

**Comment by n8pease** [comment]
@TallJimbo this replaces ci_hsc_gen2/bin.src/installExternalData.py I thought we wanted it turned into a butler subcommand?

**Comment by TallJimbo** [comment]
That's actually a Gen2 butler operation.  I think the equivalent in Gen3 is just an invocation of `butler import`.

File: python/lsst/daf/butler/script/installExternalData.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import os
+from lsst.daf.persistence import Butler
```
Comment (timj) [inline python/lsst/daf/butler/script/installExternalData.py:24]: We are shutting down this PR but the red flag here is that daf_butler can not depend on daf_persistence for anything.

**Pull request closed without merging**